<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player</class>
 <widget class="QWidget" name="Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #22222e</string>
  </property>
  <widget class="QListWidget" name="music_list">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>20</y>
     <width>701</width>
     <height>741</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget {
	color: white;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rand_button">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>530</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	background-color: #fb5b5d;
}

QPushButton:pressed {
	background-color: #fa4244;
}</string>
   </property>
   <property name="text">
    <string>Randomize it!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="play_button">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>610</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	background-color: #fb5b5d;
}

QPushButton:pressed {
	background-color: #fa4244;
}</string>
   </property>
   <property name="text">
    <string>Play it!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_button">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>690</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	background-color: #fb5b5d;
}

QPushButton:pressed {
	background-color: #fa4244;
}</string>
   </property>
   <property name="text">
    <string>Stop it!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toggle_button">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	background-color: #7fff00;
}

QPushButton:pressed {
	background-color: #006400;
}</string>
   </property>
   <property name="text">
    <string>ON</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
